js/src/react/index.js

```diff
@@ -4,8 +4,97 @@ import App from './components/App';
 import './styles.css';
 import './assets/spbc-icons.min.css';
 
-addEventListener('DOMContentLoaded', () => {
+function sanitizeJsonString(str) {
```

Comment:
We need to avoid after-processing provided data. Let's do right prepare every data during gathering it.